docs/cli/lokoctl_completion_zsh.md

```diff
@@ -0,0 +1,40 @@
+## lokoctl completion zsh
+
+Generate autocompletions script for zsh
```

Comment:
Here it's autocompletions, and below it's autocompletion. In keeping them consistent, I think it should be autocompletion here?